dev/drivers/scripts/prep/nfcens/jevs_nfcens_wave_grid2obs_prep.sh

```diff
@@ -42,7 +42,7 @@ export SENDCOM=${SENDCOM:-YES}
 export SENDECF=${SENDECF:-YES}
 export SENDDBN=${SENDDBN:-NO}
 export KEEPDATA=${KEEPDATA:-YES}
-export SENDMAIL=${SENDMAIL:-YES}
+export SENDMAIL=${SENDMAIL:-NO}
```

Comment:
@SamiraArdani-NOAA Can you please change SENDMAIL to YES in your dev driver (unless there are usually a huge number of missing file emails)? You can set SENDMAIL to NO in your J-job, if it isn't already! 

CC @malloryprow @ShelleyMelchior-NOAA